ColorPair.cpp

```diff
@@ -0,0 +1,35 @@
+#include "ColorPair.hpp"
+
+namespace TelCoColorCoder
+{
+    int getNumOfMajorColors() {
+        return sizeof(MajorColorNames) / sizeof(MajorColorNames[0]);
+    }
+
+    int getNumOfMinorColors() {
+        return sizeof(MinorColorNames) / sizeof(MinorColorNames[0]);
+    }
+
+    ColorPair GetColorFromPairNumber(int pairNumber) {
+        int zeroBasedPairNumber = pairNumber - 1;
+        MajorColor majorColor =
+            (MajorColor)(zeroBasedPairNumber / getNumOfMinorColors());
+        MinorColor minorColor =
+            (MinorColor)(zeroBasedPairNumber % getNumOfMinorColors());
+        return ColorPair(majorColor, minorColor);
+    }
+
+    int GetPairNumberFromColor(MajorColor major, MinorColor minor) {
+        return major * getNumOfMinorColors() + minor + 1;
+    }
+
+    void showColorPairManual() {
```

Comment:
consider adding new functionality in new file. that way we minimize the impact and chance of bloating big files.